SECURITY.md

```diff
@@ -0,0 +1,27 @@
+# Security Policy
+
+## Supported Versions
+
+Security updates are only provided for last released version.
+Previous versions are not fixed.
+
+| Version | Supported          |
+| ------- | ------------------ |
+| 0.8.x   | :white_check_mark: |
+| < 0.8.0 | :x:                |
+
+## Reporting a Vulnerability
+
+If any can prevent avahi-daemon from responding or even terminating it
+Use this section to tell people how to report a vulnerability.
+If you discover a way to crash avahi-daemon via network packet or make it non-responsive,
+please do not create normal public issue. Instead report it on
+[Security](https://github.com/avahi/avahi/security) page as a vulnerability.
+Give us enough time to look into it, please do not post it on public places.
```

Comment:
I think it's too vague. It should say that the project follows 90-day disclosure policy (or whatever) and that response should be expected in a reasonable amount of time (say, a week or two). Without that it looks like it's possible to sit on bugs indefinitely. I think it should be possible to copy-paste https://github.com/google/oss-vulnerability-guide/blob/main/templates/security_policies/github_security_policy.md and adjust it a bit.